common/rabbitmq/templates/etc/_rabbitmq-custom-config.tpl

```diff
@@ -0,0 +1,15 @@
+{{ if $.Values.customConfig }}
+## RabbitMQ custom config
+## Generated by helm template etc/_rabbitmq-custom-config.tpl
+## For more information please find helm chart
+##     https://github.com/sapcc/helm-charts/tree/master/common/rabbitmq
+
+## For more information about RabbitMQ configuration please find
+##     https://www.rabbitmq.com/configure.html
+
+{{- range $k, $v := .Values.customConfig }}
+{{ if $v }}
```

Comment:
This also disallows me from setting anything to `false` or 0, right? From reading the provided link on how to configure rabbitmq and it does not look like the config can handle a string `"false"`, too.

Reply:
Correct fixed only `nil` values will be ignored 
fixed!